libs/ml/include/ml/ops/reshape.hpp

```diff
@@ -34,13 +34,28 @@ class Reshape : public fetch::ml::Ops<T>
   using ArrayType     = T;
   using SizeType      = typename ArrayType::SizeType;
   using VecTensorType = typename Ops<T>::VecTensorType;
+  using SPType        = ReshapeSaveableParams;
 
   explicit Reshape(std::vector<SizeType> new_shape)
     : new_shape_(new_shape)
   {}
+
+  explicit Reshape(SPType const &sp)
+  {
+    new_shape_ = sp.new_shape;  // todo: tests for this
```

Comment:
Please raise ticket and add TODO in correct format

example: // TODO(issue 943): Make im2col efficient using iterators